composer.json

```diff
@@ -13,17 +13,17 @@
         }
     ],
     "conflict": {
-        "doctrine/persistence": ">=2.0"
+        "doctrine/persistence": "<1.3"
     },
     "require": {
-        "php": ">=5.6",
-        "sergeil/aux-bundle": "~1.0",
-        "sergeil/expander-bundle": "~1.0"
+        "php": ">=7.1",

```

Comment:
`doctrine/persistence` require PHP v7.1 and starting from v2 `Doctrine\Common\Persistence\ManagerRegistry` was removed

Reply:
What I am saying, is that with this change I won't be able just release another 2.x package version (e.g. 2.3.2), I will have to create 3.0.0 package (because using packages that require PHP 5.6 will be a non BC change).

Let me know if you're willing to update your projects and deps to 3.0.0 version of SliExtJsIntegrationBundle package.